inference-engine/tests_deprecated/behavior/vpu/myriad_tests/vpu_get_metric_tests.cpp

```diff
@@ -96,3 +96,17 @@ TEST_F(VPUGetMetric, smoke_MyriadGetFullDeviceName) {
         ASSERT_TRUE(!act_res.empty());
     }
 }
+
+TEST_F(VPUGetMetric, smoke_MyriadGetDeviceArchitecture) {
```

Comment:
Suggest adding tests that "2.1-ma2480" returns same architecture as "2.2-ma2480". And test that "2.1-ma2480" returns different architecture as for "2.1-ma2450"